### ISSUE TITLE
Toggle menu miscategorized as link (functions as button)

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#4498]
The state of the toggle menu (collapsed/expanded) in the book preview is not communicated to assistive technologies.  
The role of the navigation button is communicated as a link. Since this control does not redirect to other pages and functions as a button, it should have a button role.

<img width="297" height="168" alt="Image" src="https://github.com/user-attachments/assets/a4396a26-2e24-4fbe-afbd-f2a6816649be" />
